### ISSUE TITLE
Bad performance using OpenJ9 jdk on M1 processor macbooks 

### ISSUE THREAD

**Issue by VincentSalewyn** [ibmruntimes/openj9-openjdk-jdk11#562]
We are using 2 jre releases for our 2 IBM SPSS Modeler releases:

- Modeler 18.3: openJDK.Hotspot.8u265-b01
- Modeler 18.4: jdk-11.0.13+8_openj9-0.29.0 , downloaded from https://github.com/ibmruntimes/semeru11-binaries/releases/tag/jdk-11.0.13%2B8_openj9-0.29.0

In MacOS M1 processor MacBook, performances for Modeler 18.4 are drastically bad so that the product can't be used (it takes hours to process what should be done in one minute..)
When we replace JRE by the one from Modeler 18.3, there is no performance issue anymore.

Upgrading to last JRE release https://github.com/ibmruntimes/semeru11-binaries/releases/tag/jdk-11.0.15%2B10_openj9-0.32.0 doesn't fix this issue.
The problem doesn't appear with Intel processor MacBooks.

Let me know if you need further details

**Comment by pshipton** [comment]
When you upgraded to 11.0.15 are you using the x86 JVM or the AArch64 JVM? I assume x86.

Pls try the AArch64 Mac JVM available with 11.0.16, although it's still an early access build.
https://github.com/ibmruntimes/semeru11-binaries/releases/tag/jdk-11.0.16%2B8_openj9-0.33.0-rc1

**Comment by VincentSalewyn** [comment]
Thanks for your reply.
yes, we tested with x86 release and I just realized yesterday that there was a specific build for AArch64 in 11.0.15 (that was not the case with 11.0.13 we shipped for Modeler 18.4).
I have no M1 mac machine on my side and the QA person that can test it is on vacation this week but i already suggested to use this AArch64 release. I will let you know later next week.

**Comment by 0xdaryl** [comment]
@VincentSalewyn : were you (or anyone else) able to test this with the native Apple Silicon build to see if the performance improved?

**Comment by VincentSalewyn** [comment]
No, I haven't a Apple Silicon machine yet. I'll let you know when i can retest it.
